NorthstarDLL/dllmain.cpp

```diff
@@ -78,7 +78,8 @@ bool LoadPlugins()
 			spdlog::info("Failed to load library {}: ", std::system_category().message(GetLastError()));
 			continue;
 		}
-		HRSRC manifestResource = FindResourceW(datafile, MAKEINTRESOURCEW(101), MAKEINTRESOURCEW(RT_RCDATA));
+		static const auto rt_rcdataw = MAKEINTRESOURCEW(10);
+		HRSRC manifestResource = FindResourceW(datafile, MAKEINTRESOURCEW(101), rt_rcdataw);
```

Comment:
Why do we cache one of these but not the other? These calls are basically free and there's no reason to do this.

Reply:
I couldn't find a way to get gcc to compile `MAKEINTRESOURCEW` on an `LPSTR` (the type of `RT_RCDATA`) through any amount of casting.